app/gui2/e2e/collapsingAndEntering.spec.ts

```diff
@@ -1,13 +1,13 @@
 import { test, type Page } from '@playwright/test'
-import os from 'os'
 import * as actions from './actions'
 import { expect } from './customExpect'
 import { mockCollapsedFunctionInfo } from './expressionUpdates'
+import { CONTROL_KEY } from './keyboard'
 import * as locate from './locate'
 
 const MAIN_FILE_NODES = 11
 
-const COLLAPSE_SHORTCUT = os.platform() === 'darwin' ? 'Meta+G' : 'Control+G'
+const COLLAPSE_SHORTCUT = `${CONTROL_KEY}+G`
```

Comment:
After some changes ago, now it's possible to read COLLAPSE shortcut from bindings.ts (used for example in https://github.com/enso-org/enso/blob/e1893b65af1774bacd4b862cff11d28886bb81c2/app/gui2/src/composables/__tests__/selection.test.ts) You may try to use it here.

Reply:
I don't think E2E tests have access to the main app source tree